integration-tests/bats/sql.bats

```diff
@@ -1027,3 +1027,54 @@ SQL
     [ "${#lines[@]}" -eq 2 ]
     [[ "$output" =~ "tmp_br" ]] || false
 }
+
+@test "sql: check constraints" {
+    dolt sql <<SQL
+CREATE table t1 (
+       a INTEGER PRIMARY KEY check (a > 3),
+       b INTEGER check (b > a)
+);
+SQL
+
+    dolt sql -q "insert into t1 values (5, 6)"
+
+    run dolt sql -q "insert into t1 values (3, 4)"
+    [ $status -eq 1 ]
+    [[ "$output" =~ "constraint" ]] || false
+
+    run dolt sql -q "insert into t1 values (4, 2)"
+    [ $status -eq 1 ]
+    [[ "$output" =~ "constraint" ]] || false
+
+    dolt sql <<SQL
+CREATE table t2 (
+       a INTEGER PRIMARY KEY,
+       b INTEGER
+);
+ALTER TABLE t2 ADD CONSTRAINT chk1 CHECK (a > 3);
+ALTER TABLE t2 ADD CONSTRAINT chk2 CHECK (b > a);
+SQL
+
+    dolt sql -q "insert into t2 values (5, 6)"
+
+    run dolt sql -q "insert into t2 values (3, 4)"
+    [ $status -eq 1 ]
+    [[ "$output" =~ "constraint" ]] || false
+
+    run dolt sql -q "insert into t2 values (4, 2)"
+    [ $status -eq 1 ]
+    [[ "$output" =~ "constraint" ]] || false
+
+    dolt sql <<SQL
+ALTER TABLE t2 DROP CONSTRAINT chk1;
```

Comment:
Maybe drop one, then check, then drop the other and check? Otherwise a future change could make it where dropping one wipes them all.

Reply:
Good call